.github/CODEOWNERS

```diff
@@ -6,28 +6,34 @@
 * @asaj @cmcewen @timmoreton
 
 # directory and file-level owners. Feel free to add to this!
+/packages/attestation-proxy/ 
 /packages/attestation-service/ @nambrot
 /packages/azure/ @codyborn
 /packages/blockchain-api/ @jmrossy @cmcewen
 /packages/celotool/ @asaj
 /packages/cli/ @mcortesi @gastonponti
 /packages/contractkit/ @mcortesi @gastonponti
 /packages/dappkit/ @cmcewen
-/packages/devutils/ @mcortesi
+/packages/dev-utils/ @mcortesi
 /packages/docs/ @timmoreton
 /packages/faucet/ @jmrossy
 /packages/helm-charts/ @nambrot @timmoreton
 /packages/metadata-crawler/ @aaitor
-/packages/mobile/ @cmcewen @jmrossy
-/packages/moonpay-auth/ @annakaz
-/packages/notification-service/ @annakaz @cmcewen
+/packages/mobile/ @jeanregisser @cmcewen @jmrossy @i1skn 
+/packages/moonpay-auth/ @annakaz @i1skn 
+/packages/notification-service/ @annakaz @cmcewen @i1skn 
 /packages/phone-number-privacy/ @jmrossy
 /packages/protocol/ @asaj @nategraf
 /packages/react-components/ @jmrossy @aaronmgdr
 /packages/reserve-site/ @aaronmgdr
 /packages/terraform-modules-public/ @tkporter
 /packages/terraform-modules/ @tkporter
+/packages/transaction-metrics-exporter/ 
 /packages/typescript/ @cmcewen
 /packages/utils/ @jmrossy
-/packages/web/ @cmcewen @aaronmgdr
+/packages/verification-pool-api/
+/packages/verifier/
+/packages/walletkit/
```

Comment:
These don't exist anymore, why add them back in here?

Reply:
ah oops they directories still existed on my machine

Reply:
ditto for transaction-metrics-exporter